assembl/static2/js/app/components/common/tree/treeItem.jsx

```diff
@@ -136,14 +138,18 @@ class Child extends React.PureComponent<Props, State> {
     );
   };
 
+  expandCollapseHandler = (event: SyntheticEvent<HTMLDivElement>, expanded: boolean) => {
+    if (expanded) {
```

Comment:
return expanded ? this.scrollToElement() : this.expandCollapse(event); 
if you want

Reply:
Can't do it here @MilanBarande since `this.expandCollapse(event)` should be run every time and it doesn't depend on the value of expanded. But good remark.